tutorials/phasorpy_introduction.py

```diff
@@ -327,8 +328,49 @@
 # %%
 # Select phasor coordinates
 # -------------------------
+#
+# The :py:mod:`phasorpy.cursors` module provides functions for selecting phasor
+# coordinates to define and mask regions of interest within the phasor space.
+#
+# Mask regions of interest in the phasor space using circular cursors:
 
-# TODO
+from phasorpy.color import CATEGORICAL
+from phasorpy.cursors import mask_from_circular_cursor
+
+cursors_real = [0.69, 0.59]
+cursors_imag = [0.32, 0.33]
+radius = [0.05, 0.05]
```

Comment:
Btw, I changed this to be a list because later `radius` was re-defined as a list. Mypy does not like changing types. Should we consider also checking the tutorials with Mypy in CI even though we avoid typing in tutorials?

Reply:
Oh, I thought this was to showcase that different cursors can have different radius, which I thought it was a good thing to show (although in this case both cursors have the same radius).

Regaring checking tutorials with Mypy in CI, maybe we can discuss this in the next meeting. It can be nice to have a check in case something changes and we forget to change it in the tutorials, as in this case. But I don't know the impact of running this check in CI without typing the tutorials.